Packs/Malware/Scripts/AppendindicatorFieldWrapper/AppendindicatorFieldWrapper_test.py

```diff
@@ -0,0 +1,53 @@
+import demistomock as demisto
+import pytest
+
+
+def test_run_append_indicator_field_script():
```

Comment:
We are missing a mock for `execute_command` no?

Reply:
Added that, thanks.